<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font-family:century gothic;
	font-size:13px;
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>900</width>
      <height>620</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/background/background.jpeg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>-50</y>
      <width>481</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/logo/piranhaLogo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>691</width>
      <height>451</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe MDL2 Assets</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 8pt &quot;Segoe MDL2 Assets&quot;;</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="HomeTab">
     <attribute name="title">
      <string>HOME</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>210</y>
        <width>501</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="scan">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>210</y>
        <width>112</width>
        <height>31</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	color:white;
	background:#0492C2;
	border-radius:0px;
}</string>
      </property>
      <property name="text">
       <string>SCAN</string>
      </property>
     </widget>
     <widget class="QLabel" name="enterURL">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>131</width>
        <height>19</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
	font-style:bold;
	color:#333;
	font-size:14px;
}</string>
      </property>
      <property name="text">
       <string>ENTER URL</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>260</y>
        <width>611</width>
        <height>23</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="value">
       <number>35</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
      <property name="textDirection">
       <enum>QProgressBar::TopToBottom</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>310</y>
        <width>611</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 italic 10pt &quot;Arial&quot;;
</string>
      </property>
      <property name="text">
       <string>STATUS: The Website is a Phishing URL.</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="BlackListTab">
     <attribute name="title">
      <string>BLACKLIST</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>651</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="blacklistLabel">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>121</width>
        <height>19</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
	font-family:century gothic;
	color:#333;
	font-size:20px;
	
}</string>
      </property>
      <property name="text">
       <string>BLACKLIST</string>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
     <widget class="QPushButton" name="scan_2">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>20</y>
        <width>112</width>
        <height>31</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	color:white;
	background:#0492C2;
	border-radius:0px;
}</string>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="AboutUSTab">
     <attribute name="title">
      <string>ABOUT US</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../../Downloads/pirahnav2.0/piranhaLogo.qrc"/>
  <include location="../../../Downloads/pirahnav2.0/background.qrc"/>
 </resources>
 <connections/>
</ui>
